examples/svelte-kit/src/hooks.server.ts

```diff
@@ -0,0 +1,14 @@
+import type { Handle } from '@sveltejs/kit'
+import cookie from 'cookie'
+import { provideTelefuncContext } from 'telefunc'
+import 'telefunc/async_hooks'
```

Comment:
Because we recommend `MostlyMutations`, we don't need `telefunc/async_hooks` anymore.